paasta_itests/paasta_serviceinit.feature

```diff
@@ -43,6 +43,7 @@ Feature: paasta_serviceinit
      When we run the marathon app "test-service.main"
       And we wait for it to be deployed
      Then paasta_serviceinit status -vv for the service_instance "test-service.main" exits with return code 0 and the correct output
+      And paasta_serviceinit status -s "test-service" -i "main,test" has the correct output for instance main and exits with non-zero return code for instance test
```

Comment:
Can you add a test above to match the "good" case? Cause a python stack trace will also cause a non-zero exit code:(

    And paasta_serviceinit status -s "test-service" -i "main" has the correct output for instance main and exits with 0 and the correct output